pkg/queuesystem/queue_system.go

```diff
@@ -0,0 +1,19 @@
+package queuesystem
+
+// Messages is the base struct for getting
+// data from a specified queue system
+type Messages interface {
+	GetQueueMessages() ([]QueueMessage, error)
+	DeleteMessageFromQueue(string) error
+}
+
+// QueueMessage is the abstraction for formatting
+// and using data that comes from the queues
+type QueueMessage struct {
```

Comment:
Need your input on what data Gogeta should store from Gamebuildr to pass onto Mr.Robot